src/components/UpgradeTableOfContents.tsx

```diff
@@ -1,47 +1,10 @@
 import React from "react"
-import { motion } from "framer-motion"
-import { Link } from "gatsby"
-import styled from "@emotion/styled"
+import { Box, Link, ListItem, UnorderedList } from "@chakra-ui/react"
```

Comment:
Am I correct in thinking that we import `Link` directly from Chakra as our `Link` component is already extending it?